rpcd/etc/openstack_deploy/user_variables.yml

```diff
@@ -62,3 +62,20 @@ ceph_mons: >
   {% endfor -%}
   {% endif -%}
   {{ _var }}
+
+# locations for the package builder to look for files that contain pip packages and git repos to build from
```

Comment:
Why did all this repo build stuff need to be done for liberty? The commit message doesn't give an explanation.